src/maliput_malidrive/builder/builder_tools.cc

```diff
@@ -280,10 +280,10 @@ const std::map<LaneTravelDirection::Direction, std::string> xodr_to_maliput_dire
 
```

Comment:
This `xodr_to_maliput_direction` dictionary maps xodr value to maliput value.
Does it make sense to still have undefined? Is it possible now to have an undefined value from the xodr?